src/components/NewSubtask.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useContext } from "react";
```

Comment:
I can't seem to find the scss file that styles this new subtask component, have you created one?